raiden/tests/unit/transfer/mediated_transfer/test_mediatorstate.py

```diff
@@ -1492,7 +1491,7 @@ def test_mediator_receive_lock_expired_after_secret_reveal():
         state_change=secret_reveal,
         channelidentifiers_to_channels=channels.channel_map,
         pseudo_random_generator=pseudo_random_generator,
-        block_number=block_lock_expired,
+        block_number=block_lock_expired - 1,
```

Comment:
why did you change the block number by -1 here and below?

Reply:
This is basically saying, we receive a secret reveal message before the lock expires

Reply:
hmmm so the test was wrong previously